src/io/fastq.rs

```diff
@@ -773,12 +780,20 @@ IIIIIIJJJJJJ
         let expected = io::Error::new(io::ErrorKind::NotFound, "foo");
 
         assert_eq!(actual.kind(), expected.kind());
-        assert!(actual.to_string().starts_with("No such file or directory"))
+
+        #[cfg(unix)]
+        assert!(actual.to_string().starts_with("No such file or directory"));
+
+        #[cfg(windows)]
+        assert!(actual
+            .to_string()
+            .starts_with("The system cannot find the path specified"));
     }
 
     #[test]
     fn test_writer_to_file_dir_exists_returns_ok() {
-        let path = Path::new("/tmp/out.fq");
+        let file = tempfile::NamedTempFile::new().expect("Could not create temp file");
+        let path = file.path();
```

Comment:
I added a dev dependency on `tempfile`, which will do the right thing when creating temporary files in systems that are not unix-like.